roachpb/batch.go

```diff
@@ -62,8 +62,13 @@ func (ba *BatchRequest) IsRange() bool {
 
 // GetArg returns the first request of the given type, if possible.
 func (ba *BatchRequest) GetArg(method Method) (Request, bool) {
-	// TODO(tschottdorf): when looking for EndTransaction, just look at the
-	// last entry.
+	// when looking for EndTransaction, just look at the last entry.
```

Comment:
if this doesn't find EndTransaction at the last entry, it will still scan all the entries. probably can avoid that